pkg/services/rpcsrv/server.go

```diff
@@ -2936,3 +2938,47 @@ func (s *Server) Addresses() []string {
 	}
 	return res
 }
+
+func (s *Server) getRawNotaryPool(reqParams params.Params) (any, *neorpc.Error) {
+	if !s.chain.P2PSigExtensionsEnabled() {
+		return nil, neorpc.NewInternalServerError("P2PSignatureExtensions are disabled")
+	}
+	verbose, _ := reqParams.Value(0).GetBoolean()
+
+	nrp := s.coreServer.GetNotaryPool()
+	pool := nrp.GetVerifiedPoolTx()
+	if !verbose {
+		return pool, nil
+	}
+	res := result.RawNotaryPool{}
+	for txMain, list := range pool {
+		hashes := result.NotaryHashes{MainTxHash: txMain}
+		hashes.FallbackTxHashes = append(hashes.FallbackTxHashes, list...)
+		res.NotaryHashes = append(res.NotaryHashes, hashes)
+	}
+	return res, nil
+}
+
+func (s *Server) getRawNotaryRequest(reqParams params.Params) (any, *neorpc.Error) {
+	if !s.chain.P2PSigExtensionsEnabled() {
+		return nil, neorpc.NewInternalServerError("P2PSignatureExtensions are disabled")
+	}
+
+	txHash, err := reqParams.Value(0).GetUint256()
+	if err != nil {
+		return nil, neorpc.ErrInvalidParams
+	}
+	nrp := s.coreServer.GetNotaryPool()
+	tx, ok := nrp.TryFindTransaction(txHash)
+	if !ok {
+		return nil, neorpc.ErrUnknownTransaction
+	}
+
+	if v, _ := reqParams.Value(1).GetBoolean(); v {
+		res := result.TransactionOutputRaw{
+			Transaction: *tx,
+		}
```

Comment:
`TransactionOutputRaw` isn't relevant for notary transactions, because notary transactions are not supposed to be accepted to the chain directly (unlike the mempooled ones). Just return the transaction without any additional wrapper.

Reply:
Fixed.